app/models/king.rb

```diff
@@ -1,23 +1,18 @@
 class King < Piece
-  # def king_moves_like_this
-    # put code here
-  # end
+
   def puts_piece_type
     puts "This is a King"
   end
 
-  def valid_move?
-    # king can move from starting point to the right one square
-
-
-
-    # king can move from starting point to the left one square
-
-
-    # king can move from starting point up one column
+  # returns a true if the target location is a legal move to go to
+  def legal_move?(row,col)
+    current_row = self.row_position
+    current_col = self.col_position
```

Comment:
Since we only use these once now we could probably get away with not assigning these piece attributes to a variable